src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -967,7 +967,7 @@ private EnvVars getEnvVars(DescribableList<NodeProperty<?>, NodePropertyDescript
      */
     private void verifyNoHeaderJunk(TaskListener listener) throws IOException, InterruptedException {
         ByteArrayOutputStream baos = new ByteArrayOutputStream();
-        connection.exec("true",baos);
+        connection.exec("echo hello > null",baos);
```

Comment:
this create a file named null, Is this the intention?

Reply:
maybe it is better to make echo "", it does not create anything and does not print anything on the terminal.

Reply:
Using `echo ''` does not work as it produces output on both linux and windows.
The linux output is only a newline while the windows output would be `''` with a newline.

I will try some other commands I have in mind and will come back to you later.